pkg/assembler/backends/inmem/hasSLSA_test.go

```diff
@@ -78,9 +79,11 @@ func TestHasSLSA(t *testing.T) {
 				{
 					Subject: a1out,
 					Slsa: &model.Slsa{
-						BuiltBy:   b1out,
-						BuiltFrom: []*model.Artifact{a2out},
-						BuildType: "test type",
+						BuiltBy:    b1out,
+						BuiltFrom:  []*model.Artifact{a2out},
+						BuildType:  "test type",
+						StartedOn:  &time.Time{},
```

Comment:
Yeah, we shouldn't need these in the expected output if they are nil in the input.

Reply:
they are not nil. Changing them to:

```
func nilToZeroValue(inputTime *time.Time) time.Time {
	if inputTime == nil {
		return time.Time{}
	}
	return *inputTime
}
```